### PR TITLE
Adds bank account security prefs

### PR DESCRIPTION
**Pull request by SonixApache**
I got tired of having to set my account security to Zero every round, so this.
Tested, seemed to run just fine, SQLite database and everything. May need a quick review to see if I'm not overwriting anything important.
Special thanks to R0b0 for helping me debug, and WhiteHusky for actually PRing this PR to make it less shit

:cl:
 - rscadd: Added bank account security preferences - with help from WhiteHusky

### CODE REVIEW

**Comment by Hoshi-chan** [comment]
Cargonia thanks you.

**Comment by WhiteHusky** [comment]
Would it make more sense to store the security levels as integers in the DB instead of string?

**Comment by SonixApache** [comment]
It would, but I couldn't figure out why the input wouldn't allow 0s, so I went with strings.

**Comment by WhiteHusky** [comment]
That might be because 0 is a falsy value. I can't recall what you should do.

**Comment by WhiteHusky** [comment]
Belated but if you want to use `0` as a acceptable value, then use
```dm
if(!isnull(0))
    do_something()
```
*EDIT: Updated with the proc we seem to be using*

**Comment by WhiteHusky** [comment]
SonixApache#2

File: code/modules/client/preferences_savefile.dm [hunk 1]
```diff
@@ -715,17 +722,17 @@ AND players.player_slot = ? ;"}, ckey, slot)
 	check.Add("SELECT player_ckey FROM players WHERE player_ckey = ? AND player_slot = ?", ckey, slot)
 	if(check.Execute(db))
 		if(!check.NextRow())
-			q.Add("INSERT INTO players (player_ckey,player_slot,ooc_notes,real_name, random_name,    gender, age, species, language, flavor_text, med_record, sec_record, gen_record, player_alt_titles, disabilities, nanotrasen_relation, random_body)\
+			q.Add("INSERT INTO players (player_ckey,player_slot,ooc_notes,real_name, random_name,    gender, age, species, language, flavor_text, med_record, sec_record, gen_record, player_alt_titles, disabilities, nanotrasen_relation, bank_security, random_body)\
 			                    VALUES (?,          ?,          ?,        ?,         ?,              ?,      ?,   ?,       ?,        ?,           ?,          ?,          ?,          ?,                 ?,            ?,                   ?)",
```
Comment (SonixApache) [inline code/modules/client/preferences_savefile.dm:726]: if you're looking at this as an example, please note #19632

**Pull request merged**